mbf_abstract_nav/src/abstract_controller_execution.cpp

```diff
@@ -185,6 +186,43 @@ void AbstractControllerExecution::setVelocityCmd(const geometry_msgs::TwistStamp
   // TODO so there should be no loss of information in the feedback stream
 }
 
+bool AbstractControllerExecution::checkVelocityIgnore(const geometry_msgs::Twist& cmd_velocity)
+{
+  bool ignored_exceed_tolerance = false;
+
+  bool robot_stopped = robot_info_.isRobotStopped(1e-3, 1e-3);
+
+  // compute linear and angular velocity magnitude
+  const double cmd_linear = std::hypot(cmd_velocity.linear.x, cmd_velocity.linear.y);
+  const double cmd_angular = std::abs(cmd_velocity.angular.z);
+
+  bool cmd_is_not_zero = (cmd_linear > 0.01 || cmd_angular > 0.01);
```

Comment:
S: this is forcing u to make a double negation on the if below  (avoid contorted logic whenever possible)